src/components/ConversationIcon.vue

```diff
@@ -31,6 +31,7 @@
 			:disable-menu="disableMenu"
 			:display-name="item.displayName"
 			:preloaded-user-status="preloadedUserStatus"
+			:url="item.avatar"
```

Comment:
I would recommend to just skip all the frontend work for now and focus on the backend path.
We can create a ticket and get a frontender to work on it afterwards.